model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -93,40 +109,50 @@ def nodes_builder(model: GraphModule,
             node_type = node.target
             if node_type == getattr:
                 node_has_activation = False
-                Logger.warning(
-                    'Pytorch model has a parameter or constant Tensor value. This can cause unexpected behaviour when '
-                    'converting the model.')
+                # technically, tensor ops like shape have no quantization issue
+                if node.args[1] not in ("shape"):
+                    Logger.warning(
+                        "Pytorch model has a parameter or constant Tensor value. This can cause unexpected behaviour when "
+                        "converting the model."
+                        f" FX call_function getattr {node.args=} {node.kwargs=}"
+                    )
         elif node.op == PLACEHOLDER:
             node_type = DummyPlaceHolder
         elif node.op == OUTPUT:
-            output_nodes += node.all_input_nodes
+            output_nodes.append((node.all_input_nodes, node.args[0]))
```

Comment:
Please add a comment explaining what `node.args[0]` is expected to be

Reply:
done